UnitTests/GitCommandsTests/Git/GitCommandHelpersTest.cs

```diff
@@ -280,6 +280,18 @@ public void GetSubmoduleNamesFromDiffTest()
 			Assert.AreEqual(status.OldCommit, "2fb88514cfdc37a2708c24f71eca71c424b8d402");
 			Assert.AreEqual(status.OldName, fileName);
 
-		}
+            // Submodule name in reverse diff, rename
+
+            text = "diff --git b/Externals/conemu-inside-b a/Externals/conemu-inside-a\nindex a17ea0c..b5a3d51 160000\n--- b/Externals/conemu-inside-b\n+++ a/Externals/conemu-inside-a\n@@ -1 +1 @@\n-Subproject commit a17ea0c8ebe9d8cd7e634ba44559adffe633c11d\n+Subproject commit b5a3d51777c85a9aeee534c382b5ccbb86b485d3\n";
+            fileName = "Externals/conemu-inside-b";
+
+            status = GitCommandHelpers.GetSubmoduleStatus(text, testModule, fileName);
+
+            Assert.AreEqual(status.Commit, "b5a3d51777c85a9aeee534c382b5ccbb86b485d3");
```

Comment:
I tend to use FluentAssertions as it reads more natural: `status.Commit.Should().Be("b5a3d51777c85a9aeee534c382b5ccbb86b485d3");`

Reply:
Personally, I prefer AreEqual and in this case it follows the style of existing test cases.
I will consider FluentAssertions if updating

Note: This is an enabler, handled in #4031, so it should maybe not added to the 2.51 milestone explicitly

Reply:
That's cool, it wasn't an action item.